pkg/protecode/protecode.go

```diff
@@ -363,10 +381,20 @@ func (pc *Protecode) PollForResult(productID int, timeOutInMinutes string) Resul
 		}
 	}
 
-	if len(response.Result.Components) == 0 || response.Result.Status == statusBusy {
+	if protecodeStillWorking(response.Result.Status) {
```

Comment:
I would vote for renaming this.

```suggestion
	if scanInProgress(response.Result.Status) {
```